pkg/daemon/ceph/client/upgrade.go

```diff
@@ -104,23 +102,30 @@ func GetAllCephDaemonVersions(context *clusterd.Context, clusterName string) (*C
 }
 
 // EnableMessenger2 enable the messenger 2 protocol on Nautilus clusters
-func EnableMessenger2(context *clusterd.Context) error {
-	_, err := context.Executor.ExecuteCommandWithOutput(false, "", "ceph", "mon", "enable-msgr2")
+func EnableMessenger2(context *clusterd.Context, clusterName string) error {
+	args := []string{"mon", "enable-msgr2"}
+	buf, err := NewCephCommand(context, clusterName, args).Run()
+	output := string(buf)
 	if err != nil {
 		return fmt.Errorf("failed to enable msgr2 protocol. %+v", err)
 	}
+	logger.Debug(output)
```

Comment:
Doesn't `NewCephCommand()` already log the command? Maybe I'm just thinking of the command and args that it logs.

Reply:
the `Run()` method does log the command but not the output which is being logged here.